sdk-java/src/main/java/io/littlehorse/sdk/wfsdk/SpawnedThread.java

```diff
@@ -4,4 +4,7 @@
  * This is the output of `ThreadBuilder::spawnThread()`. It is used as input for
  * `ThreadBuilder::waitForThread()`.
  */
-public interface SpawnedThread {}
+public interface SpawnedThread {
+
+    WfRunVariable getThreadNumberVariable();
```

Comment:
Hmm. I would prefer not exposing this to the public interface because then the customer might try to use that internal variable.